incubator/elasticsearch/README.md

```diff
@@ -69,30 +69,35 @@ The following tables lists the configurable parameters of the elasticsearch char
 | `master.persistence.storageClass`    | Master persistent volume Class                                      | `nil`                                |
 | `master.persistence.accessMode`      | Master persistent Access Mode                                       | `ReadWriteOnce`                      |
-| `data.exposeHttp`                   | Expose http port 9200 on data Pods for monitoring, etc              | `false`                              |
+| `data.name`                          | Data component name                                                 | `data`                               |
 | `data.replicas`                      | Data node replicas (statefulset)                                    | `3`                                  |
 | `data.resources`                     | Data node resources requests & limits                               | `{} - cpu limit must be an integer`  |
+| `data.podAnnotations`                | Data StatefulSet annotations                                        | `{}`                                 |
 | `data.heapSize`                      | Data node heap size                                                 | `1536m`                              |
+| `data.antiAffinity`                  | Data anti-affinity policy                                           | `soft`                               |
+| `data.exposeHttp`                    | Data expose http port 9200 on data Pods for monitoring, etc         | `false`                              |
+| `data.budget.maxUnavailable`         | Data pod disruption budget max unavailable                          | `1`                                  |
 | `data.persistence.enabled`           | Data persistent enabled/disabled                                    | `true`                               |
 | `data.persistence.name`              | Data statefulset PVC template name                                  | `data`                               |
 | `data.persistence.size`              | Data persistent volume size                                         | `30Gi`                               |
 | `data.persistence.storageClass`      | Data persistent volume Class                                        | `nil`                                |
 | `data.persistence.accessMode`        | Data persistent Access Mode                                         | `ReadWriteOnce`                      |
-| `data.podAnnotations`                | Data StatefulSet annotations                                        | `{}`                                 |
 | `data.terminationGracePeriodSeconds` | Data termination grace period (seconds)                             | `3600`                               |
-| `data.antiAffinity`                  | Data anti-affinity policy                                           | `soft`                               |
```

Comment:
it looks like there's a few documentation fixups that are unrelated to the PDBs ...  It might be a good idea to put them through as a seperate PR to reduce the scope of this PR to just the PDBs.